### PR TITLE
Op: Mirror padding

### PR DESCRIPTION
**Pull request by briangrifiin**
* Mirror padding: https://github.com/onnx/onnx/blob/master/docs/Operators.md#pad

This pull request just extends existing padding translation to support 'reflect' mode.

### CODE REVIEW

**Comment by jackwish** [comment]
Hi @briangrifiin thanks for your contribution! I will look into it later today.

File: tflite2onnx/op/padding.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 class Padding(Operator):
     TypeMapping = {
         tflite.BuiltinOperator.PAD: 'Pad',
+        tflite.BuiltinOperator.MIRROR_PAD: 'MirrorPad',
```
Comment (jackwish) [inline tflite2onnx/op/padding.py:18]: Should this be `Pad`? The value of `TypeMapping` is the op type of ONNX.

File: tflite2onnx/op/padding.py [hunk 2]
```diff
@@ -34,6 +35,9 @@ def parse(self):
         opcode = self.model.OperatorCodes(op.OpcodeIndex()).BuiltinCode()
         assert(opcode in self.TypeMapping)
 
+        if opcode is tflite.BuiltinOperator.MIRROR_PAD:
+            self.attrs['mode'] = 'reflect'
```
Comment (jackwish) [inline tflite2onnx/op/padding.py:39]: Can we make this to something like below? We prefer to setup `mode` here explicitly.

```py
if opcode is MIRROR_PAD:
    self.attrs['mode'] = 'reflect'
else:
    self.attrs['mode'] = 'constant'
```

File: tflite2onnx/op/padding.py (hunk 1 above)
Comment (jackwish) [reply tflite2onnx/op/padding.py:18]: We can also update the [`type`](https://github.com/jackwish/tflite2onnx/pull/36/files#diff-a4232bb36ec94ea0d244d786f19f1ff6d48dffc083d021bf84f95a02f47d0173R29) to use `TypeMapping`.

**Review by jackwish — CHANGES_REQUESTED** [review]
@briangrifiin looks good in general, I will approve after `TypeMapping` fixed. Thanks for your contribution!

File: tflite2onnx/op/padding.py (hunk 1 above)
Comment (briangrifiin) [reply tflite2onnx/op/padding.py]: https://github.com/jackwish/tflite2onnx/pull/36/files#diff-a4232bb36ec94ea0d244d786f19f1ff6d48dffc083d021bf84f95a02f47d0173R18

https://github.com/jackwish/tflite2onnx/pull/36/files#diff-a4232bb36ec94ea0d244d786f19f1ff6d48dffc083d021bf84f95a02f47d0173R29

File: tflite2onnx/op/padding.py (hunk 2 above)
Comment (briangrifiin) [reply tflite2onnx/op/padding.py:44]: https://github.com/jackwish/tflite2onnx/pull/36/files#diff-a4232bb36ec94ea0d244d786f19f1ff6d48dffc083d021bf84f95a02f47d0173R43-R46

**Review by jackwish — APPROVED** [review]
LGTM

**Comment by jackwish** [comment]
@briangrifiin This is now merged, thanks!

**Pull request merged**